### DISCUSSION TITLE
Transpile to VBScript?

### DISCUSSION THREAD

**Discussion by ghost** [pfusik/cito#113] [General]
VBScript is a general-purpose scripting language that is widely used among system administrators in the Microsoft environment. It's available as a part of all Windows versions unlike PowerShell that sometimes you need to install manually on older Windows versions.

**Comment by pfusik** [comment]
VBScript might have some uses for administrative tasks, but:

1. Microsoft itself is replacing it with Visual Basic .NET.
2. It is very unlikely that users of VBScript ever need any libraries transpiled from Fusion.
